homeassistant/components/electrasmart/climate.py

```diff
@@ -159,6 +160,8 @@ def __init__(self, device: ElectraAirConditioner, api: ElectraAPI) -> None:
         self._consecutive_failures = 0
         self._skip_update = True
         self._was_available = True
+        
+        SID_EXPIRATION = int(timedelta(minutes=19).total_seconds())
```

Comment:
Overriding a constant from the library is not the right way to do this.

Reply:
Awaiting for a better way to do it: https://github.com/jafar-atili/pyElectra/pull/4/files